lib/puppet/type/postgresql_conn_validator.rb

```diff
@@ -6,6 +6,7 @@
           purposes such as monitoring."
```

Comment:
The text is not rendering properly in REFERENCE.md. Take a look at https://github.com/puppetlabs/puppetlabs-firewall/blob/master/lib/puppet/type/firewall.rb for how this could be done.